### DISCUSSION TITLE
User Realm Migration

### DISCUSSION THREAD

**Discussion by marcosbarbero** [keycloak/keycloak#26102] [Ideas]
Disclaimer: I tried to find a discussion on this subject, but I couldn't

-----
Recently, I came across a use case where I had to migrate accounts from one realm to another, but I couldn't find an easy way to do it using the UI. The only way I could make it work was by manipulating the database.

What if we build a feature where we can select a given realm to move to an entirely new one?

**Comment by marcosbarbero** [comment]
I wouldn't mind contributing to adding this feature if the team believes it's valuable.

**Reply by Hosea-kibet** [reply]
I would not  mind assisting on this.

**Reply by marcosbarbero** [reply]
Hi @Hosea-kibet, thanks for engaging! Maybe we should just start working on it and open an MR. I am not sure whether it will ever be merged, but after 11 months without any interaction from the maintainers, I'm not so positive they ever will.

**Reply by cuberinooo** [reply]
Interesting idea ! Any news on this ? I also could assist on that. We have a use case where it would be great by doing this via API.

**Reply by marcosbarbero** [reply]
@cuberinooo nothing so far :D
